backend/src/v5/models/revisions.js

```diff
@@ -64,14 +64,16 @@ Revisions.getRevisionByIdOrTag = (teamspace, model, revision, projection = {}) =
 	model, { $or: [{ _id: revision }, { tag: revision }] }, projection);
 
 Revisions.updateRevisionStatus = async (teamspace, project, model, revision, status) => {
-	const query = { $or: [{ _id: revision }, { tag: revision }] };
-	const res = await db.updateOne(teamspace, collectionName(model), query, { $set: { void: status } });
+	const res = await db.findOneAndUpdate(teamspace, collectionName(model),
```

Comment:
@ChristosTsiotsias  can you write an e2e test for this change?

i.e. a revision update where the user provided a rev tag.